app/src/main/java/com/infomaniak/mail/ui/main/menu/MoveViewModel.kt

```diff
@@ -49,20 +49,21 @@ class MoveViewModel @Inject constructor(
     private val messageUid inline get() = savedStateHandle.get<String?>(MoveFragmentArgs::messageUid.name)
     private val threadsUids inline get() = savedStateHandle.get<Array<String>>(MoveFragmentArgs::threadsUids.name)!!
 
+    var currentFolderId: String? = null
     var filterResults: MutableLiveData<List<Folder>> = MutableLiveData()
 
     fun cancelSearch() {
         filterJob?.cancel()
     }
 
-    fun getFolderIdAndCustomFolders() = liveData(ioCoroutineContext) {
+    fun getFolders() = liveData(ioCoroutineContext) {
 
-        val folderId = messageUid?.let { messageController.getMessage(it)!!.folderId }
+        currentFolderId = messageUid?.let(messageController::getMessage)?.folderId
             ?: threadController.getThread(threadsUids.first())!!.folderId
 
-        val customFolders = folderController.getCustomFolders().getCustomMenuFolders()
+        val folders = folderController.getMoveFolders().getCustomMenuFolders()
```

Comment:
`getCustomMenuFolders()` doesn't only return the list of custom folders so maybe we can rename it with a better name